healme.lic

```diff
@@ -439,8 +439,11 @@ class HealMe
   end
 
   def get_cast_command_for_wound(wound)
-    # Default, the cast command must specify a body part.
-    return 'cast skin' unless wound
+    unless wound
+      # Default, the cast command must specify a body part.
+      echo "No longer aware of a wound healable by the prepared spell. Casting on skin just to discharge harness and cambrinth."
```

Comment:
Should this use DRC.message?

Reply:
I'm fine either way. "echo" won't have any extra emphasis to it. DRC.message will make it yellow and stand out. What's your preference?

Reply:
## Example using "echo"
![image](https://user-images.githubusercontent.com/68095633/115815605-dfde5080-a3bc-11eb-8bec-810ffe37fc5c.png)

## Example using DRC.message
![image](https://user-images.githubusercontent.com/68095633/115815806-44011480-a3bd-11eb-853c-a7f88b625646.png)

Reply:
I'm fine with either. Just wasn't sure if it was common for people to use message or echo. Agree that it's probably not worth drawing attention to - as the user doesn't care.